### ISSUE TITLE
Support --remove-orphans with docker-compose create

### ISSUE THREAD

**Issue by Wilfred** [docker/compose#5175]
It would be nice if `docker-compose create` supported the `--remove-orphans` argument.

Currently, it doesn't seem to be possible to create containers without starting, whilst also removing orphans. I can't see a command (or combination of commands) that do this.

**Comment by Wilfred** [comment]
One option would be to do:

```
$ docker-compose down --remove orphans
$ docker-compose create
```

However, if anyone has started a container that I've created, this will stop them.

**Comment by shin-** [comment]
`create` is deprecated in favor of `up` using the `--no-start` flag.

**Comment by Wilfred** [comment]
@shin- has this landed / been released? I can't see `--no-start` in 1.16.1.

**Comment by shin-** [comment]
Ah, not yet. It will be in 1.17.0 ! https://github.com/docker/compose/pull/5142
